features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/introspect.jag

```diff
@@ -1,33 +1,44 @@
 <%
 
     var log = new Log("Jaggery service for token introspection");
+
+    var userInfoEndpoint = "https://localhost:8243/userinfo";
     var introspectEndpoint = "https://localhost:9443/oauth2/introspect"
     var tokenP1 = request.getCookie("WSO2_AM_TOKEN_1_Default").value;
     var tokenP2 = request.getCookie("AM_ACC_TOKEN_DEFAULT_P2").value;
+    var token = tokenP1 + tokenP2;
+    var userData = {};
+    var user_result = get(userInfoEndpoint, userData , {
+        "Authorization": "Bearer "+token
+    });
+    log.info(user_result);
```

Comment:
better make this a debug log

Reply:
resolved by https://github.com/wso2/carbon-apimgt/pull/6241/